### PR TITLE
Added jscs grunt task

### LINKED ISSUE #5: Integrate JSCS
Check out http://jscs.info/ for more information! We'd be glad to help with integration if necessary!

### PR DESCRIPTION
**Pull request by ptte**
Added jscs with a very very minimum default styleguide.

We should look at what styleguide is a good default for us. I tried jquery, but it's very different from how we write code.

- added install script
- added task under option `noJSCS`

Fixes #5

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -19,8 +19,10 @@
   "homepage": "https://github.com/bloglovin/lintlovin",
   "dependencies": {
     "dependency-check": "^2.1.0",
+    "grunt": "^0.4.5",
```
Comment (voxpelli) [inline package.json]: As Grunt is added as a perrDependency to Lintlovin (and a devDependency) it shouldn't be here as well

Comment (ptte) [inline package.json]: I was about to say that I didn't add it, but the diff says I did. I think it was automatically added by the npm install --save command... *sigh*

**Comment by voxpelli** [comment]
Apart from above comment this looks good to me! :+1: Feel to merge once that dependency is removed

**Comment by mikesherov** [comment]
Thanks for adding! A few notes:

You should name the config file `.jscsrc`. JSCS looks for it by default so you don't even have to specify the config file location to grunt, and if people are using an editor plugin for linting, it'll pick it up as well.

Lastly, it looks like the line added to package.json is a case of mixed spaces and tabs. I just chuckled at the irony there.

Let me know if you have further questions about JSCS!

**Comment by voxpelli** [comment]
@mikesherov Thanks for the pointers! Just pushed some additional tweaks for eg `.jscsrc` – and I also added `package.json` to our `.editorconfig` check so any whitespace errors there should now be detected (couldn't find a tab in there now though).

**Comment by voxpelli** [comment]
Also published a new version as `1.10.0`

**Pull request merged**